src/blocks/focused_window.rs

```diff
@@ -177,12 +184,13 @@ impl ConfigBlock for FocusedWindow {
             })
             .expect("failed to start watching thread for `window` block");
 
-        let text = TextWidget::new(id, 0, shared_config);
+        let text = TextWidget::new(id, 0, shared_config).with_text("x");
```

Comment:
Why do we need this fallback?

Reply:
I just took it from whatever block I was looking at as a reference to add format support. Removed.